nexus/ocp-config-oauth/dc.yml

```diff
@@ -0,0 +1,89 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: nexus3
+objects:
+- apiVersion: v1
+  kind: DeploymentConfig
+  metadata:
+    labels:
+      app: nexus3
+    name: nexus3
+  spec:
+    replicas: 1
+    selector:
+      app: nexus3
+      deploymentconfig: nexus3
+    strategy:
+      activeDeadlineSeconds: 21600
+      resources: {}
+      recreateParams:
+        timeoutSeconds: 600
+      type: Recreate
+    template:
+      metadata:
+        labels:
+          app: nexus3
+          deploymentconfig: nexus3
+      spec:
+        containers:
+        - args:
+            - '--https-address=:8443'
+            - '--provider=openshift'
+            - '--openshift-service-account=nexus-oauth'
+            - '--upstream=http://localhost:8081'
+            - '--tls-cert=/etc/tls/private/tls.crt'
+            - '--tls-key=/etc/tls/private/tls.key'
+            - '--cookie-secret=SECRET'
```

Comment:
`SECRET`?

Reply:
Hi @michaelsauter. Will answer your question in the evening.